models/candidates_2017.js

```diff
@@ -2,18 +2,212 @@
 
 module.exports = function(sequelize, DataTypes) {
   return sequelize.define('candidates_2017', {
-    id: {
-      type: DataTypes.INTEGER(10).UNSIGNED.ZEROFILL,
-      allowNull: false,
-      primaryKey:true,
-      autoIncrement: true
-    },
+      AadharId: {
```

Comment:
suppose some candidate want specialized fields for every application, like references for the job of associate prof and assistant prof in another subject may be different.